lib/matplotlib/tests/test_axes.py

```diff
@@ -4974,23 +4974,30 @@ def test_bar_single_height():
     ax.bar(0, 1, bottom=range(4), width=1, orientation='horizontal')
 
 
-def invalid_axes_limits():
+def test_invalid_axes_limits():
+    error_string = 'NaN or Inf cannot be the argument values'
     with pytest.raises(ValueError) as err:
-        plt.set_xlim(left=np.nan)
+        plt.xlim(left=np.nan)
+    assert err.value.message == error_string
```

Comment:
Do exceptions have this `message` attribute in py3k? The test suite is failing for py3k, saying that it doesn't (but it works for python27).